src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrPatchHandler.cs

```diff
@@ -361,10 +361,16 @@ public async Task ApplyPatch(SourceMapping mapping, VmrIngestionPatch patch, Can
         string patchPath,
         CancellationToken cancellationToken)
     {
+        var files = new List<string>();
+        if (_fileSystem.GetFileInfo(repoPath).Length == 0)
+        {
+            _logger.LogDebug("Patch {patch} is empty. Skipping file enumeration..", patchPath);
+            return files;
+        }
+
         var result = await _processManager.ExecuteGit(repoPath, new string[] { "apply", "--numstat", "--allow-empty", patchPath }, cancellationToken);
```

Comment:
why not also remove the `--allow-empty` argument?

Reply:
Yeah, good point